.github/workflows/run-linters.yaml

```diff
@@ -0,0 +1,49 @@
+name: TF/Shell Linters
+on:
+  push:
+    brainches:
+      - main
+  pull_request:
+    types:
+      - opened
+      - edited
+      - reopened
+      - synchronize
+jobs:
+  tflint:
+  # from https://github.com/marketplace/actions/setup-tflint
+    name: Lint TF
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        name: Checkout source code
+
+      - uses: actions/cache@v4
+        name: Cache plugin dir
+        with:
+          path: ~/.tflint.d/plugins
+          key: ${{ matrix.os }}-tflint-${{ hashFiles('.tflint.hcl') }}
+
+      - uses: terraform-linters/setup-tflint@v4
+        name: Setup TFLint
+        with:
+          tflint_version: v0.52.0
```

Comment:
either have spaces or no spaces around between the lines - we haven't been too consistent across files but let's try to be consistent in the same file